### PR TITLE
[RegistrationTest] Add RegistrationServiceImplTest class

### PR DESCRIPTION
**Pull request by makrushinoff**
Add class RegistrationServiceImplTest. Later it is going to be updated, when Validator will be written.

### CODE REVIEW

File: src/test/java/ua/com/sportfood/services/RegistrationServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import org.springframework.security.crypto.password.PasswordEncoder;
+import ua.com.sportfood.dao.CustomerDAO;
+import ua.com.sportfood.formRegistration.RegistrationForm;
+import ua.com.sportfood.models.Customer;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+public class RegistrationServiceImplTest {
```
Comment (MLGMAG) [inline src/test/java/ua/com/sportfood/services/RegistrationServiceImplTest.java:18]: Remove public modifier due to sonar rule:
https://rules.sonarsource.com/java/RSPEC-5786

"JUnit5 test classes and methods should have default package visibility"

**Review by MLGMAG — CHANGES_REQUESTED** [review]

File: src/test/java/ua/com/sportfood/services/RegistrationServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    private RegistrationForm registrationForm;
+
+    @Mock
+    private PasswordEncoder passwordEncoder;
```
Comment (MLGMAG) [inline src/test/java/ua/com/sportfood/services/RegistrationServiceImplTest.java:23]: Remove redundant mock

**Review by MLGMAG — CHANGES_REQUESTED** [review]

File: src/test/java/ua/com/sportfood/services/RegistrationServiceImplTest.java (hunk 1 above)
Comment (makrushinoff) [reply src/test/java/ua/com/sportfood/services/RegistrationServiceImplTest.java]: Done

File: src/test/java/ua/com/sportfood/services/RegistrationServiceImplTest.java (hunk 2 above)
Comment (makrushinoff) [reply src/test/java/ua/com/sportfood/services/RegistrationServiceImplTest.java]: Done

**Review by MLGMAG — APPROVED** [review]

**Review by bagachvlad — APPROVED** [review]

**Pull request merged**